Examples/Run/Reconstruction/Common/SeedingExample.cpp

```diff
@@ -136,6 +136,7 @@ int runSeedingExample(int argc, char* argv[],
   seedingCfg.beamPosX = 0;
   seedingCfg.beamPosY = 0;
   seedingCfg.impactMax = 3.;
+  seedingCfg.requireMonotonicZ = true;
```

Comment:
In effect, this turns it on for all geometries and not only the generic detector. However, all other seedingCfg are hardcoded,including the B-field value along Z, so this does not worsen the situation. The real solution would be to make the seeding example configurable at runtime, but that's outside the scope of this PR. Still I just wanted to point this out for posterity.